src/main/java/core/basesyntax/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.Operation;
+import java.util.HashMap;
+import java.util.Map;
+
+public class OperationStrategy {
+    Map<Operation, OperationHandler> operationStrategyrMap;
+    {
+        operationStrategyrMap = new HashMap<>();
+        operationStrategyrMap.put(Operation.BALANCE, new BalanceOperationHandler());
+        operationStrategyrMap.put(Operation.PURCHASE, new PurchaseHandler());
+        operationStrategyrMap.put(Operation.RETURN, new ReturnHandler());
+        operationStrategyrMap.put(Operation.SUPPLY, new SupplyHandler());
+    }
```

Comment:
let's init it via Map.of() in Main and pass here to the constructor for more flexibility